packages/core/src/components/document-component/document-component.tsx

```diff
@@ -225,12 +250,18 @@ export class DocumentComponent implements ComponentInterface {
           </div>
         </div>
         <slot name="annotation" />
-        {this.open && (this.inhoud || this.gereserveerd) && (
+        {this.open && (this.inhoud || this.gereserveerd || this.vervallen) && (
```

Comment:
Waarom wordt er gebruik gemaakt van nederlandse property namen? Er zijn met dit component opeens een heleboel.

Reply:
Ja, ik zat hier vreselijk in dubio.

Het component "Document Component" is een presentatie component voor een "Document Component" van Ozon. Ozon is een Web API waar juridische documenten uitkomen. Dit zijn hele grote documenten. Ze zijn opgeknipt in "Document Componenten". Deze document componenten zijn volgens een standaard genaamd "CIMOW".

In deze standaard worden nagenoeg uitsluitend Nederlandse termen gebruikt. Omdat het toolkit component zo dicht bij de CIMOW standaard zit leek het mij duidelijker om deze property names als "domein specifiek" te bestempelen. Domein specifieke termen vertalen we niet.